modules/apps/commerce/commerce-product-content-search-web/src/main/java/com/liferay/commerce/product/content/search/web/internal/display/context/CPPriceRangeFacetsDisplayContext.java

```diff
@@ -132,25 +130,16 @@ public boolean isCPPriceRangeValueSelected(
 			String fieldName, String fieldValue)
 		throws PortalException {
 
-		Optional<String[]> parameterValuesOptional =
+		return ArrayUtil.contains(
 			_portletSharedSearchResponse.getParameterValues(
-				fieldName, _renderRequest);
-
-		if (parameterValuesOptional.isPresent()) {
-			String[] parameterValues = parameterValuesOptional.get();
-
-			return ArrayUtil.contains(parameterValues, fieldValue);
-		}
-
-		return false;
+				fieldName, _renderRequest),
+			fieldValue);
 	}
 
 	public boolean isShowClear(String fieldName) {
-		Optional<String[]> parameterValuesOptional =
+		return ArrayUtil.isNotEmpty(
```

Comment:
@julschong This is again a non-consistent conversion. `isPresent()` only does null check, so it returns `true` if the array is empty in the original logic.